Gemfile

```diff
@@ -65,7 +65,7 @@ gem 'haml'
 gem 'redcarpet'
 gem 'sass'
 gem 'sass-rails'
-gem 'truncate_html', '0.9.2'
```

Comment:
I just copied it from Spree when I brought all gem requirements from spree_core.
So the question is why did spree lock it at this version.

Anyway 093 is the latest so we can just remove the version nunber from the gemfile:
https://rubygems.org/gems/truncate_html

Reply:
I dug a bit further and it seems that we don't use this gem in OFN. It used in Spree in https://github.com/openfoodfoundation/spree/blob/2-1-0-stable/core/app/helpers/spree/orders_helper.rb to truncate the product description. We don't because we should it in the product modal, and there's no trace of that helper method in our codebase nor I see the gem used at all.

Reply:
I went with https://github.com/openfoodfoundation/openfoodnetwork/pull/6362